sdk/storage_blobs/Cargo.toml

```diff
@@ -38,7 +38,7 @@ oauth2 = { version = "4.0.0", default-features = false }
 mock_transport = { path = "../../eng/test/mock_transport" }
 
 [features]
-default = ["enable_reqwest", "azure_identity/default"]
```

Comment:
`azure_identity` is a dev-dependency, so this doesn't work. It fails when publishing.